src/store/options.js

```diff
@@ -174,35 +169,34 @@ const Options = {
           // sendMessage may fail without potential target
         });
 
-      sync(options, keys).catch(() => null);
+      sync(options, keys);
 
       return options;
     },
     observe: (_, options, prevOptions) => {
+      observer.run(options, prevOptions);
+
       // Sync if the current memory context get options for the first time
       if (!prevOptions) sync(options);
-
-      idleOptionsObservers = (async () => {
-        for (const fn of observers) {
-          try {
-            await fn(options, prevOptions);
-          } catch (e) {
-            console.error(`[options] Error while observing options: `, e);
-          }
-        }
-      })();
     },
   },
 };
 
+export default Options;
+
 chrome.runtime.onMessage.addListener((msg) => {
   if (msg.action === UPDATE_OPTIONS_ACTION_NAME) {
     store.clear(Options, false);
     store.get(Options);
   }
 });
 
-export default Options;
+export function isPaused(options, domain = '') {
```

Comment:
Can we please do not move functions that are not affected by the PR? Keeping them in the same files makes reviewing easier and keeps git history meaningful.

Reply:
I completely reorganized the file, and the final structure should be like this. It is pointless to make another PR with moving lines around one file.

Reply:
it's just a function order, why it makes a difference? is it alphabetical now?

Reply:
The export is made just after the definition of the model. It keeps things more connected.